### PR TITLE
Add Preemption to Glossary page

### PR DESCRIPTION
**Pull request by daminisatya**
#5993 

I need a review on the content and the tags which I added for preemption, please!

Thank you.

### CODE REVIEW

File: content/en/docs/reference/glossary/Preemption.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Preemption
+id: Preemption
+date: 2019-01-31
+full_link: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#preemption
+short_description: >
+  Preemption logic in Kubernetes helps a pending pod find a suitable node by deleting low priority pods existing on that node.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/Preemption.md]: pod -> Pod
pods -> Pods

File: content/en/docs/reference/glossary/Preemption.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- operation
+---
+ Preemption logic in Kubernetes helps a pending pod find a suitable node by deleting low priority pods existing on that node.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/Preemption.md]: pod -> Pod
pods -> Pods

File: content/en/docs/reference/glossary/Preemption.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Preemption
+id: Preemption
+date: 2019-01-31
+full_link: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#preemption
+short_description: >
+  Preemption logic in Kubernetes helps a pending Pod find a suitable node by deleting low priority Pods existing on that node.
```
Comment (neolit123) [inline content/en/docs/reference/glossary/Preemption.md]: we can slight change this sentence.

>  Preemption logic in Kubernetes helps a pending Pod to find a suitable Node by evicting low priority Pods existing on that node.

sorry i didn't catch this earlier.
[1]

File: content/en/docs/reference/glossary/Preemption.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- operation
+---
+ Preemption logic in Kubernetes helps a pending Pod find a suitable node by deleting low priority Pods existing on that node.
+
```
Comment (neolit123) [inline content/en/docs/reference/glossary/Preemption.md]: [1]

**Comment by daminisatya** [comment]
@neolit123 I addressed the changes and squashed the commits.

**Comment by daminisatya** [comment]
@Rajakavitha1 Thank you for the review :) I updated the doc as pointed. Please let me know if I need to add any more in the preemption docs.

**Comment by cody-clark** [comment]
Thanks for all of your contributions, @daminisatya!

Preemption Preview: https://deploy-preview-12460--kubernetes-io-master-staging.netlify.com/docs/concepts/configuration/pod-priority-preemption/#preemption

Glossary Preview: https://deploy-preview-12460--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?all=true#term-Preemption

/approve

**Pull request merged**